sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/RecognizeInvoicesOptions.java

```diff
@@ -0,0 +1,108 @@
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer.models;
+
+import com.azure.core.annotation.Fluent;
+
+import java.time.Duration;
+
+import static com.azure.ai.formrecognizer.implementation.Utility.DEFAULT_POLL_INTERVAL;
+
+/**
+ * Options that may be passed when using recognize invoice APIs on Form Recognizer client.
+ */
+@Fluent
+public final class RecognizeInvoicesOptions {
+    private FormContentType contentType;
+    private boolean includeFieldElements;
+    private String locale;
+    private Duration pollInterval = DEFAULT_POLL_INTERVAL;
+
+    /**
+     * Get the type of the form. Supported Media types including .pdf, .jpg, .png or .tiff type file stream.
+     *
+     * @return the {@code contentType} value.
+     */
+    public FormContentType getContentType() {
+        return contentType;
+    }
+
+    /**
+     * Get the boolean which specifies if to include form element references in the result.
+     *
+     * @return the {@code includeFieldElements} value.
+     */
+    public boolean isFieldElementsIncluded() {
+        return includeFieldElements;
+    }
+
+    /**
+     * Get the duration between each poll for the operation status. If none is specified, a default of
+     * 5 seconds is used.
+     *
+     * @return the {@code pollInterval} value.
+     */
+    public Duration getPollInterval() {
+        return pollInterval;
+    }
+
+    /**
+     * Set the type of the form. Supported Media types including .pdf, .jpg, .png or .tiff type file stream.
+     *
+     * @param contentType the provided form content type.
+     *
+     * @return the updated {@code RecognizeInvoicesOptions} value.
+     */
+    public RecognizeInvoicesOptions setContentType(final FormContentType contentType) {
+        this.contentType = contentType;
+        return this;
+    }
+
+    /**
+     * Set the boolean which specifies if to include form element references in the result.
+     *
+     * @param includeFieldElements the boolean to specify if to include form element references in the result.
+     *
+     * @return the updated {@code RecognizeInvoicesOptions} value.
+     */
+    public RecognizeInvoicesOptions setFieldElementsIncluded(final boolean includeFieldElements) {
+        this.includeFieldElements = includeFieldElements;
+        return this;
+    }
+
+    /**
+     * Set the duration between each poll for the operation status. If none is specified, a default of
+     * 5 seconds is used.
+     *
+     * @param pollInterval the duration to specify between each poll for the operation status.
+     *
+     * @return the updated {@code RecognizeInvoicesOptions} value.
+     */
+    public RecognizeInvoicesOptions setPollInterval(final Duration pollInterval) {
+        this.pollInterval = pollInterval == null ? DEFAULT_POLL_INTERVAL : pollInterval;
+        return this;
+    }
+
+    /**
+     * Get the locale information for the invoice.
+     * Supported locales include: en-AU, en-CA, en-GB, en-IN, en-US.
```

Comment:
Paul said that invoices would only support en-US for preview.2

Reply:
ohh I missed this. Where did he comment that? then the swagger needs to be updated for invoice so it only allows that value, no?

Reply:
Oh I see, then would we not be using the `Locale` enum for receipts/bc/invoices in that case since they differ in their support?
cc: @maririos

Reply:
First comment on this Teams [chat](https://teams.microsoft.com/l/message/19:88f2d9dea2344075b5d4bc34d82a3d1c@thread.skype/1604010293331?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1604010293331&teamName=Azure%20SDK&channelName=Service%20-%20Form%20Recognizer&createdTime=1604010293331). Personally I see us handling this like we do with the FormContentType / lack of image/bmp support for analyze custom form. Use Locale, but document the allowed values. Let service handle failure. Once other locales are supported, this will only be a docs update.

Reply:
Hmm I'm not sure about this. I think it will be better to keep this one separated and have it support only en-US so that it doesn't confuse our customers. This is one of those cases that locales will be different per endpoint so better to have enums per endpoint than generic ones.

Reply:
Curious why would treat this one different than FormContentType? Also feel free to ignore me, this will be a non-issue in python since we will use strings only and I shouldn't sway the decision for other languages

Reply:
(this conversation is not a blocker for the current approach of the PR, so feel free to merge while we keep discussing this).

So in the arch board meeting, at least for .NET, the decision was to have independent extensible enums per endpoint for locale and language. In this case, in .NET I am going to create Locale per endpoint. I am also advocating for changing the swagger to have each locale enum defined in each endpoint.

 Note that in the invoice case, if you send something different than `en-US` the service will fail. I believe that for `FormContentType` that is not the case.

Reply:
> I believe that for FormContentType that is not the case.

I actually don't know what the service does here, because we raise an error in the generated code about image/bmp not being supported. Last I tried in canary, it worked with bmp.... so I guess they will be removing that functionality before deploying to prod.